### PR TITLE
Add Consul ECS v0.5 release notes

### PR DESCRIPTION
**Pull request by sanon-dev**
### Description
Adding Consul ECS release notes for v0.5.x to docs

### PR Checklist

* [x] ~updated test coverage~
* [x] external facing docs updated
* [x] not a security concern

### CODE REVIEW

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Consul ECS release notes for version 0.5.x
+---
+
+# Consul ECS 0.5.x
+
+## Release Highlights
+
+- **Audit Logging (Enterprise) :** This feature supports audit logging by providing an `audit-logging` flag, providing a log of authenticated events if enabled.
+
+- **Support Consul AWS IAM auth method :** This feature provides support obtaining ACL tokens from Consul's AWS IAM auth method.
```
Comment (cthain) [inline website/content/docs/release-notes/consul-ecs/v0_5_x.mdx:14]:
```suggestion
- **AWS IAM Auth Method :** This feature provides support for Consul's AWS IAM auth method. This allows AWS IAM roles and users to authenticate with Consul to obtain ACL tokens.
```

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- **Mesh Gateways :** This feature introduces Consul WAN federation via mesh gateways ECS tasks, allowing for Consul servers in different datacenters to be federated.
```
Comment (cthain) [inline website/content/docs/release-notes/consul-ecs/v0_5_x.mdx:16]:
```suggestion
- **Mesh Gateways :** This feature introduces support for running mesh gateways as ECS tasks. Mesh gateways enable service mesh communication across datacenter and admin partition boundaries.
```

**Review by cthain — COMMENTED** [review]
Nice work on this AJ. I left a few suggestions in my review.

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx (hunk 2 above)
Comment (trujillo-adam) [reply website/content/docs/release-notes/consul-ecs/v0_5_x.mdx:16]: I think we can link to the **Configure the gateway task module** topic in the installation page so readers know where to go for usage instructions.

**Review by trujillo-adam — COMMENTED** [review]
A couple of minor suggestions to help readers find the related information.

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx (hunk 1 above)
Comment (sanon-dev) [reply website/content/docs/release-notes/consul-ecs/v0_5_x.mdx]: Added new phrasing to match @cthain suggestion and linked to appropriate section for configuration info in [latest commit](https://github.com/hashicorp/consul/pull/14010/commits/d5f334cf1ee381d5d5b60a4bc33d8f5f99ce9f17)

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx (hunk 2 above)
Comment (sanon-dev) [reply website/content/docs/release-notes/consul-ecs/v0_5_x.mdx]: Added new phrasing to match @cthain suggestion and linked to appropriate section for usage info in [latest commit](https://github.com/hashicorp/consul/pull/14010/commits/d5f334cf1ee381d5d5b60a4bc33d8f5f99ce9f17)

File: website/content/docs/release-notes/consul-ecs/v0_5_x.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+page_title: 0.5.x
+description: >-
+  Consul ECS release notes for version 0.5.x
+---
+
+# Consul ECS 0.5.x
+
+## Release Highlights
+
+- **Audit Logging (Enterprise) :** This feature supports audit logging by providing an `audit-logging` flag, providing a log of authenticated events if enabled.
```
[the remark this answers is not in the document]
Comment (sanon-dev) [reply website/content/docs/release-notes/consul-ecs/v0_5_x.mdx]: Suggestion added in [latest commit](https://github.com/hashicorp/consul/pull/14010/commits/d5f334cf1ee381d5d5b60a4bc33d8f5f99ce9f17)

[the remark this answers is not in the document]
Comment (sanon-dev) [reply website/content/docs/release-notes/consul-ecs/v0_5_x.mdx]: Deferred to @trujillo-adam's [comment](https://github.com/hashicorp/consul/pull/14010#discussion_r940402804) in reference to Consul docs being not the best for this description.

**Review by cthain — APPROVED** [review]
LGTM

**Pull request merged**